pkg/api/types.go

```diff
@@ -3822,4 +3822,10 @@ const (
 	// When the --failure-domains scheduler flag is not specified,
 	// DefaultFailureDomains defines the set of label keys used when TopologyKey is empty in PreferredDuringScheduling anti-affinity.
 	DefaultFailureDomains string = metav1.LabelHostname + "," + metav1.LabelZoneFailureDomain + "," + metav1.LabelZoneRegion
+
+	// "kube-system" is the default lock object namespace
+	DefaultLockObjectNamespace string = NamespaceSystem
+
+	// "kube-scheduler" is the default lock object name
+	DefaultLockObjectName = "kube-scheduler"
```

Comment:
these really seems like they belong in the component config package, with names that scope them to the scheduler